EDU-757: Update Certificate Info in Docs (#2187)
temporalio/documentation: docs-src/cloud-context/certificates-intro.md

id: certificates-intro
title: How to manage certificates in Temporal Cloud
sidebar_label: Manage certificates
description: Certificates needed for Temporal Cloud and Worker Processes
tags:
  - guide-context
---

> Temporal uses several certificates for secure access and communication.

Reviewer: ```suggestion
Temporal Cloud requires security certificates for secure access and communication.
```

> Temporal Cloud requires security certificates for secure access and communication.

Reviewer 2: ```suggestion
[Temporal Cloud](https://temporal.io/cloud) requires security certificates for secure access and communication.
```

> [Temporal Cloud](https://temporal.io/cloud) access is secured by the mutual Transport Layer Security (mTLS) protocol, which requires a CA certificate from the user.

Reviewer 2: ```suggestion
Temporal Cloud access is secured by the mutual Transport Layer Security (mTLS) protocol, which requires a CA certificate from the user.
```

[Worker Processes](/workers/#worker-process) require CA certificates and private keys to connect to Temporal Cloud.
Temporal Cloud does not require an exchange of secrets; only the certificates produced by private keys are used for verification.

> All certificates used by Temporal Cloud must meet the provided requirements.

Reviewer 2: ```suggestion
All certificates used by Temporal Cloud must meet the following requirements.
```
